symbols/arm9.yml

```diff
@@ -3054,6 +3054,28 @@ arm9:
         NA: 0x201A9E4
         JP: 0x201AA3C
       description: "Note: unverified, ported from Irdkwia's notes"
+    - name: InitAnimationControl
+      address:
+        EU: 0x201C0EC
+      description: |-
+        Initialize the animation_control structure
+        
+        r0: animation_control
+    - name: InitAnimationControlWithSet
+      address:
+        EU: 0x201C14C
+      description: |-
+        Initialise the animation_control structure, and set a certain value in a bitflag to 1
+        
+        r0: animation_control
+    - name: InitAnimationControlWithSetClone
+      address:
+        EU: 0x201C168
+      description: |-
+        Identical to InitAnimationControlWithSet
```

Comment:
By "identical", do you mean all the instructions are exactly the same? If so, we should just give `InitAnimationControlWithSet` a second address, i.e.

```yml
    - name: InitAnimationControlWithSet
      address:
        EU:
          - 0x201C14C
          - 0x201C168
```

Reply:
They are. Will fix that.
(curious part is that they are adjacent. I’ll presume one of them was a variant of the other at some point and the devs then made them the same function. And the compiler didn’t optimized that).

Reply:
> (curious part is that they are adjacent. I’ll presume one of them was a variant of the other at some point and the devs then made them the same function. And the compiler didn’t optimized that).

I've seen this in a couple other places too. This seems like a plausible explanation of how it could happen, though I don't know for sure.